src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -356,6 +359,12 @@ private void LoadPublishTestResultsInputs(IExecutionContext context, Dictionary<
                 _runTitle = string.Empty;
             }
 
+            eventProperties.TryGetValue(PublishTestResultsEventProperties.TestRunSystem, out _testRunSystem);
+            if (_testRunSystem == null)
+            {
+                _testRunSystem = string.Empty;
```

Comment:
default value should be VSTS instead of empty ? check with PM once.

Reply:
Yeah, but that will come from the caller vsts-task-lib api call. https://github.com/Microsoft/vsts-task-lib/pull/373